src/components/Header/index.tsx

```diff
@@ -297,30 +266,34 @@ export default function Header() {
           >
             {t('header.farm')}
           </StyledNavLink>
-          <StyledExternalLink id="aurora-bridge" href={'https://rainbowbridge.app/transfer'}>
-            Rainbow Bridge <span style={{ fontSize: '11px' }}>↗</span>
-          </StyledExternalLink>
-          <StyledExternalLink id="terra-bridge" href={'https://app.allbridge.io/bridge'}>
-            Bridge from Terra <span style={{ fontSize: '11px' }}>↗</span>
-          </StyledExternalLink>
+          <StyledHomeNavLink id={`home-link`} to={'/'}>
+            {'Trisolaris'}
```

Comment:
nit does this need to be a string?
```suggestion
            Trisolaris
```